### PR TITLE
[Swagger] Doc: Changing the name of a definition

### PR DESCRIPTION
**Pull request by lyrixx**
ref: https://github.com/api-platform/core/pull/1607

### CODE REVIEW

File: core/swagger.md [hunk 1]
```diff
@@ -179,3 +179,50 @@ Will produce the following Swagger documentation:
   }
 }
 ```
+
+### Changing the name of a definition
+
+ApiPlatform generates a definition name based on the serializer `groups` defined
+in the (`de`)`denormalization_context`. It's possible to override the name
```
Comment (alanpoulain) [inline core/swagger.md]: (`de`)`normalization_context`

File: core/swagger.md [hunk 2]
```diff
@@ -179,3 +179,50 @@ Will produce the following Swagger documentation:
...
+ *          },
+ *      },
+ * )
+ */
+class User
+{
+}
+```
+
+It's also possible to re-use the `normalization_context`:
```
Comment (alanpoulain) [inline core/swagger.md]: (`de`)`denormalization_context`

**Comment by lyrixx** [comment]
Thanks @alanpoulain . I have applied your recommendations

**Comment by lyrixx** [comment]
Thanks ;)

**Comment by dunglas** [comment]
Thanks @lyrixx!

**Pull request merged**